### ISSUE TITLE
What versions of Bootstrap are supported?

### ISSUE THREAD

**Issue by arielpontes** [davidstutz/bootstrap-multiselect#963]
I'm using Bootstrap 4 and the plugin just doesn't work. When I click the menu the class `open` is added to `<div class="btn-group">` and `aria-expanded="true"` is added to the button element but the dropdown still doesn't show up. If I manually uncheck the `display: none` property on the `<ul class="multiselect-container dropdown-menu">`, the dropdown shows but the styling is broken.

If this plugin does not support Bootstrap 4, this should be added to the documentation.

**Comment by gurubobnz** [comment]
I'm using it with Bootstrap 4 and it appears to be working, other than the styling is broken. Admittedly my use of it is very plain (no plugins, no fancy stuff) but it is working.

**Comment by davidstutz** [comment]
Bootstrap 4 is not really supported yet, but on my list for the next changes - updating to Bootstrap 4 shouldn't be much work. Apart from that there is a Bootstrap 2.x branch.

**Comment by davidstutz** [comment]
Also see #741.

**Comment by davidstutz** [comment]
Will close this issue and leave #741 open as there were some suggestions.

**Comment by tiesont** [comment]
@arielpontes Bootstrap 4 (Beta) uses the `show` class rather than `open` (see the [Usage](http://getbootstrap.com/docs/4.0/components/dropdowns/#usage) section). I would imagine that this library would have to add both classes if @davidstutz wanted to be compatible with both Bootstrap 3 and Bootstrap 4, or use the VERSION property of the dropdown plugin to determine which class to use.
